Footer/footer.css

```diff
@@ -0,0 +1,52 @@
+/**footer**/
+.style-footer {
```

Comment:
If possible, try not to change font-families after you have declared in `:root`. If it's the only component that needs it, then it is fine ..!